### ISSUE TITLE
Getting larger loss on retrain

### ISSUE THREAD

**Issue by rahulnirdhar** [allanzelener/YAD2K#151]
I'm  training model using given retrain script with underwater dataset data downloaded from here.

On  last epoch I got loss: 20, and it gives wrong bounding boxes.[give prediction about other classes]

also i'm trying by using just model_body without taking any weight that time I get NaN value as the loss

model_body, model = create_model(anchors, class_names, load_pretrained=False, freeze_body=False)
model.compile(optimizer='adam', loss={'yolo_loss': lambda y_true, y_pred: y_pred}) 
logging = TensorBoard()
model.fit([image_data, boxes, detectors_mask, matching_true_boxes],
              np.zeros(len(image_data)),
              validation_split=0.1,
              batch_size=8,
              epochs=100,
              callbacks=[logging])

model.save_weights('trained.h5')

when I run this I get loss = nan

1. How to reduce the loss - I'm trying by changing learning rate and batch size.
2. How can train my own model by using only yolov2 architecture[without any weight]
